dev/com.ibm.ws.install.featureUtility/src/com/ibm/ws/install/featureUtility/FeatureUtilityExecutor.java

```diff
@@ -0,0 +1,60 @@
+/*******************************************************************************n * Copyright (c) 2019 IBM Corporation and others.n * All rights reserved. This program and the accompanying materialsn * are made available under the terms of the Eclipse Public License v1.0n * which accompanies this distribution, and is available atn * http://www.eclipse.org/legal/epl-v10.htmln *n * Contributors:n *     IBM Corporation - initial API and implementationn *******************************************************************************/
+package com.ibm.ws.install.featureUtility;
+
+
+import java.io.IOException;
+import java.util.List;
+import java.util.logging.Level;
+
+import com.ibm.ws.install.InstallException;
+import com.ibm.ws.install.InstallKernel;
+import com.ibm.ws.install.InstallKernelFactory;
+import com.ibm.ws.install.internal.InstallKernelImpl;
+import com.ibm.ws.kernel.boot.cmdline.Arguments;
+import com.ibm.ws.kernel.feature.internal.cmdline.ArgumentsImpl;
+
+/**
+ *
+ */
+public class FeatureUtilityExecutor {
```

Comment:
Are we able to follow how we do for InstallUtilityExecutor to handle the command options, help, and actions (install and download)

Reply:
Yes i believe that is the plan, the current cli is just basic enough to get featureUtility working